src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,135 @@
+import * as R from 'react';
+import * as RRD from 'react-router-dom';
+
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person, PersonSort } from '../types';
+import { getPeople } from '../api';
+
+export const PeoplePage: R.FC = () => {
+  const [people, setPeople] = R.useState<Person[]>([]);
+  const [isLoading, setIsLoading] = R.useState(true);
+  const [errMsg, setErrMsg] = R.useState('');
+  // const { pathname, search } = RRD.useLocation();
+
+  const [searchParams] = RRD.useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries');
+  const reverseKoeff = searchParams.get('order') === 'desc' ? -1 : 1;
+  const sortField = (searchParams.get('sort') || '') as PersonSort;
+
+  R.useEffect(() => {
+    setErrMsg('');
+
+    getPeople()
+      .then((peopleFromServer: Person[]) => {
+        peopleFromServer.forEach((p: Person, i: number, arr: Person[]) => {
+          const person = arr[i];
+
+          person.father = arr.find((perent) => perent.name === p.fatherName);
+          person.mother = arr.find((perent) => perent.name === p.motherName);
+        });
+
+        setPeople(peopleFromServer);
+      })
+      .catch(() => setErrMsg('Something went wrong'))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  let visiblePeople = people;
+
+  if (sex) {
+    visiblePeople = visiblePeople.filter((p) => p.sex === sex);
+  }
+
+  if (centuries.length) {
+    visiblePeople = visiblePeople.filter((p) => {
+      return centuries.includes(Math.ceil(p.born / 100).toString());
+    });
+  }
+
+  if (query) {
+    const toLowerQuery = query.toLocaleLowerCase();
+
+    visiblePeople = visiblePeople.filter((p) => {
+      return `${p.name} ${p.fatherName || ''} ${p.motherName || ''}`
+        .toLocaleLowerCase()
+        .includes(toLowerQuery);
+    });
+  }
+
+  switch (sortField) {
+    case PersonSort.NAME:
+      visiblePeople = visiblePeople
+        .sort((a, b) => (
+          reverseKoeff * a[sortField].localeCompare(b[sortField])
+        ));
+      break;
+
+    case PersonSort.BORN:
+    case PersonSort.DIED:
+      visiblePeople = visiblePeople
+        .sort((a, b) => (
+          reverseKoeff * (a[sortField] - b[sortField])
+        ));
+
+      break;
+
+    case PersonSort.NONE:
+    default:
```

Comment:
missed actions for default case 
```suggestion
    default:
      break;
```